packages/react-components/react-combobox/src/components/Listbox/Listbox.test.tsx

```diff
@@ -87,31 +87,36 @@ describe('Listbox', () => {
     );
 
     const firstOption = getByTestId('firstOption');
-    expect(getByTestId('listbox').getAttribute('aria-activedescendant')).toBeNull();
-
-    fireEvent.focus(getByTestId('listbox'));
-
     expect(getByTestId('listbox').getAttribute('aria-activedescendant')).toEqual(firstOption.id);
   });
 
-  it('should set active option to first selected option on focus', () => {
+  it('should set active option to first selected option if one is selected', () => {
     const { getByTestId } = render(
-      <Listbox data-testid="listbox" selectedOptions={['Blue', 'Green']}>
+      <Listbox data-testid="listbox" selectedOptions={['Green']}>
```

Comment:
Changed this test b/c we actually don't want to set initial focus to the selected items in multiselect cases (that logic was already in combobox & dropdown, I think this test on listbox just snuck through)